network-store-server/src/main/resources/db/changelog/changesets/changelog_20250415T064500Z.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd" logicalFilePath="db/changelog/changesets/changelog_2022-09-26T06:45:00Z.xml">
+    <changeSet author="lesoteti" id="1664174319767-1">
```

Comment:
You don't delete column cgmescontrolareas ?
You want to do it later ?
But when I create a study, areas are still imported in cgmescontrolareas. Should be in area and areaboundary tables ?

Reply:
just forgot

Reply:
added